packages/api/cms-api/src/dam/images/images.controller.ts

```diff
@@ -95,7 +103,7 @@ export class ImagesController {
     }
 
     @DisableCometGuards()
-    @Get(`/:hash/${smartImageUrl}`)
+    @Get(`/:hash/:contentHash?/${smartImageUrl}`)
```

Comment:
Have you tried if this also works for non-preview URLs (i.e. in the site)?

Reply:
Here I tested all endpoints with and without the new contentHash parameter:

https://github.com/user-attachments/assets/055d55c3-1814-42cd-8e0a-a94c87882fab